styles.css

```diff
@@ -433,3 +435,161 @@ li {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 600px) {
+  .menu {
+    display: none;
+  }
+
+  .navigation {
```

Comment:
- [ ] In the mobile version the nav list should be hidden It should be shown in only the desktop version. Please fix it
### Your design
![his](https://user-images.githubusercontent.com/81727988/213683916-25f54791-c0d0-483c-892c-23d4e2bc741d.jpg)